frontend/src/components/blocks/BlockDetails.tsx

```diff
@@ -14,151 +15,175 @@ export interface Props {
-                  </Term>
-                }
-                imgLink="/static/images/icon-m-filter.svg"
-                text={block.gasPrice.toLocaleString()}
-              />
-            </Col>
-            <Col md="3">
-              <CardCell
-                title={
-                  <Term title={"Status"}>
-                    {"Current status of the block (finalizing or finalized). "}
-                    <a
-                      href={"https://docs.near.org/docs/interaction/rpc#block"}
-                    >
-                      docs
-                    </a>
-                  </Term>
-                }
-                imgLink="/static/images/icon-t-status.svg"
-                text={block.isFinal ? "Finalized" : "Finalizing..."}
-              />
-            </Col>
-          </Row>
-          <Row noGutters className="border-0">
-            <Col md="4">
-              <CardCell
-                title={
-                  <Term title={"Created"}>
-                    {"Timestamp of when this block finalized. "}
-                  </Term>
-                }
-                text={moment(block.timestamp).format(
-                  "MMMM DD, YYYY [at] h:mm:ssa"
-                )}
-                className="block-card-created border-0"
-              />
-            </Col>
-            <Col md="8">
-              <CardCell
-                title={
-                  <Term title={"Hash"}>
-                    {"Unique identifier (hash) of this block. "}
-                  </Term>
-                }
-                text={block.hash}
-                className="border-0"
-              />
-            </Col>
-          </Row>
+    <SubConsumer>
+      {(context) => (
+        <>
           <Row noGutters>
-            <Col md="12">
-              <CardCell
-                title={
-                  <Term title={"Parent Hash"}>
-                    {"Unique identifier (hash) of previous block. "}
-                  </Term>
-                }
-                text={
-                  block.prevHash === "11111111111111111111111111111111" ? (
-                    "Genesis"
-                  ) : (
-                    <BlockLink blockHash={block.prevHash}>
-                      {block.prevHash}
-                    </BlockLink>
-                  )
-                }
-                className="block-card-parent-hash border-0"
-              />
+            <Col className="block-info-container">
+              <Row noGutters className="block-info-header">
+                <Col md="3">
+                  <CardCell
+                    title={
+                      <Term title={"Transactions"}>
+                        {"Number of transactions in this block. "}
+                        <a
+                          href={
+                            "https://docs.nearprotocol.com/docs/concepts/transaction"
+                          }
+                        >
+                          docs
+                        </a>
```

Comment:
Note, we need to update `docs.nearprotocol.com` with `docs.near.org`

Reply:
let's make this a new PR